app/routers/event.py

```diff
@@ -25,16 +27,64 @@ async def eventedit(request: Request):
                                       {"request": request})
 
 
-@router.get("/view/{id}")
-async def eventview(request: Request, id: int):
+@router.get("/view/{event_id}")
+async def eventview(request: Request, event_id: int,
+                    db: Session = Depends(get_db)):
+    try:
+        event = get_event_by_id(db, event_id)
+    except NoResultFound:
+        raise HTTPException(status_code=404, detail="Event not found")
+    except MultipleResultsFound:
+        raise HTTPException(status_code=500, detail="Multiple events found")
+    start_format = '%A, %d/%m/%Y %H:%M'
```

Comment:
למה יש גם את שם החודש וגם את מספר החודש?

Reply:
@sagizaidor %A is the is the week day, %m is the month and %M is minutes. Am I missing something?

Reply:
אוי...תכלס, טעות שלי, התבלבלתי עם %B,...